### PR TITLE
[SD-2707] Correct the display of the error reported by macro

### PR DESCRIPTION
**Pull request by marwoodandrew**

### CODE REVIEW

File: client/app/scripts/superdesk-authoring/authoring.js [hunk 1]
```diff
@@ -1592,8 +1592,8 @@
                                 authoringWorkspace.close();
                             }
                         }, function(err) {
-                            if (err.data._issues['validator exception']) {
```
Comment (sivakuna-aap) [inline client/app/scripts/superdesk-authoring/authoring.js]: We still need this. In other cases we receive error message in validator exception

Comment (marwoodandrew) [inline client/app/scripts/superdesk-authoring/authoring.js]: According to Tolga the validation exception is not thrown any more!

**Comment by sivakuna-aap** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Comment by plamut** [comment]
Is there a test for this? :smile_cat:

**Pull request merged**